### PR TITLE
string-manipulation-3 | Angel Leung

### PR DESCRIPTION
**Pull request by angelsleung**
image

### CODE REVIEW

File: string-manipulation-3/src/is-anagram.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-prototype-builtins */
+/* exported isAnagram */
+function isAnagram(firstString, secondString) {
```
Comment (thebearingedge) [inline string-manipulation-3/src/is-anagram.js:3]: Very creative 👍

File: string-manipulation-3/src/title-case.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/* exported titleCase */
+function titleCase(string) {
```
Comment (thebearingedge) [inline string-manipulation-3/src/title-case.js:2]: Short and sweet 😄

**Review by thebearingedge — APPROVED** [review]
Bravissima 👏

File: string-manipulation-3/src/is-anagram.js (hunk 1 above)
Comment (angelsleung) [reply string-manipulation-3/src/is-anagram.js:3]: Thank you!! 😄 🙌

**Pull request merged**